android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -385,9 +384,22 @@ private void setupPurchases(String apiKey, String appUserID, @Nullable Boolean o
             if (useAmazon != null && useAmazon) {
                 store = Store.AMAZON;
             }
-            CommonKt.configure(this.applicationContext, apiKey, appUserID, observerMode,
-                    platformInfo, store, new DangerousSettings(),
-                    shouldShowInAppMessagesAutomatically, verificationMode);
+            PurchasesAreCompletedBy purchasesAreCompletedBy;
+            if (observerMode != null) {
+                purchasesAreCompletedBy = observerMode ?
+                        PurchasesAreCompletedBy.MY_APP : PurchasesAreCompletedBy.REVENUECAT;
+            } else {
+                purchasesAreCompletedBy = PurchasesAreCompletedBy.REVENUECAT;
```

Comment:
Should we just puss null in this case? Seems like PHC accepts null and then we wouldn't be forced to pass the default again here.